config/application.rb

```diff
@@ -51,5 +51,21 @@ class Application < Rails::Application
     config.exceptions_app = self.routes
 
     config.active_job.queue_adapter = :sidekiq
+
+    # Rails HTML Sanitizer
+    # Allow HTML tags & attributes that aren't allowed by default in the SanitizeHelper
+
+    # Rails defaults
+    default_sanitize_tags = ActionView::Base.sanitized_allowed_tags
+    default_sanitize_attributes = ActionView::Base.sanitized_allowed_attributes
+
+    # App's desired tags & attributes
+    desired_sanitize_tags = %w[table thead tbody tfoot tr th td]
+    desired_sanitize_attributes = %w[colspan scope]
+
+    # Merge default_sanitize_tags with desired_sanitize_tags
+    config.action_view.sanitized_allowed_tags = default_sanitize_tags.merge(desired_sanitize_tags).to_a
+    config.action_view.sanitized_allowed_attributes = default_sanitize_attributes.merge(desired_sanitize_attributes).to_a
```

Comment:
I'd say that this logic should go in something more tied to the component table helper. Aside from anything else if this went into a gem then you would be reliant on the installer to do this themselves as part of setup.

Possibly in the `railtie` itself or an initializer for it.

Reply:
looking at this [SO thread](https://stackoverflow.com/questions/53482734/add-custom-tags-to-sanitized-allowed-tags-without-replacing-defaults) regarding not overriding defaults too

Reply:
During local development, these allowed tags were done inline within the helper
`concat(govuk_table_row { sanitize(table_row, tags: %w(table thead tbody tfoot tr th td)) })`

Would this be suffficient @jsugarman

Reply:
if this is the only place it is currently needed then I think this fits the bill for now. If it impacts other helpers then it could get moved out to an initializer at a later stage. 

Be good to understand why it is needed for this helper and not others too?!